docs/commands/Invoke-Pester.mdx

```diff
@@ -106,9 +106,9 @@ with 'Util' and their subdirectories.
 
 ```powershell
 $config = [PesterConfiguration]@{
-    Should = @{ # \<- Should configuration.
+    Should = @{ # <- Should configuration.
 
-        ErrorAction = 'Continue' # \<- Always run all Should-assertions in a test
+        ErrorAction = 'Continue' # <- Always run all Should-assertions in a test
```

Comment:
@bravo-kernel Is the current escaping of `<` intentional? If so, we should just remove the arrow in the example so we don't have to fix this every time